modules/lightning_features/lightning_core/src/Routing/RouteSubscriber.php

```diff
@@ -59,27 +57,6 @@ public function alterRoutes(RouteCollection $collection) {
   }
 
   /**
-   * Checks if we are currently viewing an entity at its canonical route.
-   *
-   * @param \Drupal\Core\Entity\EntityInterface $entity
-   *   The entity.
-   * @param \Drupal\Core\Routing\RouteMatchInterface|NULL $route_match
-   *   (optional) The current route match.
-   *
-   * @return bool
-   *   TRUE if we are at the entity's canonical route, FALSE otherwise.
-   */
-  public static function isViewing(EntityInterface $entity, RouteMatchInterface $route_match = NULL) {
-    $route_match = $route_match ?: \Drupal::routeMatch();
```

Comment:
This was superseded by the new event-based Quick Edit access stuff, elsewhere in this PR.